docker/metadata/docker-compose.yml

```diff
@@ -117,17 +115,13 @@ services:
       mysql:
         condition: service_healthy
     networks:
-      app_net:
-        ipv4_address: 172.16.240.13
-    extra_hosts:
-      - "localhost:172.16.240.10"
-      - "elasticsearch:172.16.240.11"
+      - app_net
     healthcheck:
       test: [ "CMD", "curl", "-f", "http://localhost:8586/healthcheck" ]
 
   ingestion:
     container_name: openmetadata_ingestion
-    image: openmetadata/ingestion:0.12.0
+    image: openmetadata/ingestion:local
```

Comment:
This is the file we use when doing `metadata docker --start`, will be the user able to pull `openmetadata/ingestion:local`?

Reply:
Ah! Good catch Thanks! This was done for testing the local builds! I will revert the changes.

Reply:
Fixed this :)